data-tools/data/can_data.json5

```diff
@@ -123,7 +123,7 @@
     },
     {
       id: 12,
-      number: "G99XXX5",
+      number: "G99XXX7",
```

Comment:
@tdonaworth is this an intentional change ?

Reply:
Only to prove data was being changed, so yes.